src/baseSelectors.js

```diff
@@ -2,10 +2,26 @@
 import { createSelector } from 'reselect';
 
 import { allPrivateNarrowStr } from './utils/narrow';
-import { getAllMessages } from './directSelectors';
+import { getAllMessages, getPresence } from './directSelectors';
 import { NULL_ARRAY } from './nullObjects';
+import config from './config';
 
 export const getPrivateMessages = createSelector(
   getAllMessages,
   messages => messages[allPrivateNarrowStr] || NULL_ARRAY,
 );
+
+export const getValidPresence = createSelector(getPresence, presences => {
```

Comment:
`baseSelectors` is intended for the very simplest getters.
This should be somewhere else.

Also I am not a fan of the name - not descriptive enough.
Having Date.now() in the selector is also problematic, as it makes it non-pure.
We should discuss that further.